app/helpers/application_helper.rb

```diff
@@ -90,7 +90,7 @@ def link_to_my_learning(_module_item, link_args = { class: 'govuk-link, govuk-!-
   end
 
   def clear_flash
-    flash[:alert] = nil
+    flash[:error] = nil
```

Comment:
In order for the request specs to pass we need this moved to the `ApplicationController` which is a change I have already made in my refactoring.

Reply:
See #250 for potential rebase